### ISSUE TITLE
Sunstone: No way for enable graphic console for created VMs

### ISSUE THREAD

**Issue by kvaps** [OpenNebula/one#1789]
# Bug Report
## Version of OpenNebula

- [ ] 5.2.2
- [ ] 5.4.0
- [ ] 5.4.1
- [ ] 5.4.2
- [ ] 5.4.3
- [ ] 5.4.4
- [ ] 5.4.5
- [X] 5.4.6
- [ ] 5.4.7
- [ ] Development build

## Component

- [ ] Authorization (LDAP, x509 certs...)
- [ ] Command Line Interface (CLI)
- [ ] Contextualization
- [ ] Documentation
- [ ] Federation and HA
- [ ] Host, Clusters and Monitorization
- [ ] KVM
- [ ] Networking
- [ ] Orchestration (OpenNebula Flow)
- [ ] Packages
- [ ] Scheduler
- [ ] Storage & Images
- [X] Sunstone
- [ ] Upgrades
- [ ] User, Groups, VDCs and ACL
- [ ] vCenter

## Description
No way for set variable `GRAPHIC/PORT` through sunstone interface for already created VMs.

### Expected Behavior
Field for input `GRAPHIC/PORT` number or calculate port automatically if graphic enabled and PORT is not set.

### Actual Behavior
Nothing happend VNC is not working because `GRAPHIC/PORT` is not set

## How to reproduce
I used Template with:
```
GRAPHICS = [
  LISTEN = "0.0.0.0",
  RANDOM_PASSWD = "YES",
  TYPE = "VNC" ]
```
* Create VM, from this remplate
* Then power off this VM
* Update configuration --> Set graphic to none
* Update configuration --> Set graphic to VNC

# Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by abelCoronado93** [comment]
Related to #1774

**Comment by abelCoronado93** [comment]
Solved in #1774
